services/docsrs/docsrs.service.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  build_status: Joi.boolean().required(),
+}).required()
```

Comment:
The API returns an array of objects that contain the `build_status` field, not a single object, so this schema will need to be adjusted accordingly. 

Maybe try something like this?

```suggestion
const schema = Joi.array()
  .items(
    Joi.object({
      build_status: Joi.boolean().required(),
    })
  )
  .min(1)
  .required()
```